timestring/Date.py

```diff
@@ -55,9 +55,39 @@
 )
 
 
+class Specified:
+    def __init__(self, year, month, day, daytime, hour, minute, second, microsecond):
+        self.year = year
+        self.month = month
+        self.day = day
+        self.daytime = daytime
```

Comment:
is daytime not included in hour, min, sec? Do we need this separately?

Reply:
"9 o'clock" does not specify daytime -- am or pm is inferred but not explicit.